src/main/java/kitchenpos/order/dto/OrderRequest.java

```diff
@@ -0,0 +1,48 @@
+package kitchenpos.order.dto;
+
+import kitchenpos.order.domain.Order;
+import kitchenpos.order.domain.OrderLineItem;
+import kitchenpos.ordertable.domain.OrderTable;
+
+import java.util.List;
+
+public class OrderRequest {
+    private Long orderTableId;
+    private String orderStatus;
```

Comment:
이미 `OrderStatus`라는 `Enum`을 잘 정의를 해 놓으셨으니 
`Request`에서도 바로 받아도 될 것 같습니다 😄 
이렇게 하면 `orderStatus`에서 허용하지 않는 값 들도 자동으로 예외 처리 되니 수고를 덜 수 있겠습니다 😅 
```suggestion
    private OrderStatus orderStatus;
```

Reply:
dto에서 최대한 원시값이나 참조값으로 유지하는 게 가독성 측면에서 좋을까? 
라고 생각되어 시작했는데 enum은 있는 편이 더 깔끔하고 이뻐보이네요. 

감사합니다~!